sql/pg_variables_trans.sql

```diff
@@ -534,3 +534,39 @@ ROLLBACK;
 SELECT package FROM pgv_stats() ORDER BY package;
 
 SELECT pgv_free();
+
+-- Variables should be insertable after pgv_remove
+BEGIN;
+SELECT pgv_insert('test', 'x', ROW (1::int), TRUE);
+SELECT pgv_remove('test', 'x');
+SELECT pgv_insert('test', 'x', ROW (1::int), TRUE);
+ROLLBACK;
+
+SELECT * FROM pgv_list() order by package, name;
+
+BEGIN;
+SELECT pgv_insert('test', 'x', ROW (1::int), TRUE);
+SELECT pgv_remove('test', 'x');
+SELECT pgv_insert('test', 'x', ROW (1::int), TRUE);
+COMMIT;
```

Comment:
I would change it to something like that:

```sql
BEGIN;
SELECT pgv_insert('test', 'x', ROW (1::int, 2::int), TRUE);
SELECT pgv_select('test', 'x');
SELECT pgv_remove('test', 'x');
SELECT pgv_insert('test', 'x', ROW (3::int, 4::int), TRUE);
SELECT pgv_select('test', 'x');
COMMIT;
```

In order to check that variable is actually working after deletion and we are able to change/read its value.

Reply:
Ok, it sounds reasonable to me.